<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Armadillo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/armadillo.png</normaloff>img/armadillo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_8">
   <property name="leftMargin">
    <number>2</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>2</number>
   </property>
   <property name="bottomMargin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="QSplitter" name="split_left">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QFrame" name="fr_left">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QTabWidget" name="tab_left"/>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="fr_right">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QFrame" name="fr_bottom">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QSplitter" name="split_bottom">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QSplitter" name="split_right">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QStackedWidget" name="sw_main">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
          <widget class="QTabWidget" name="tab_right"/>
         </widget>
         <widget class="QStackedWidget" name="sw_bottom">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="fr_topbar">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QFrame" name="fr_tab">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>6</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>4</number>
         </property>
         <item row="0" column="3">
          <widget class="QPushButton" name="b_closetab">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/close.png</normaloff>img/close.png</iconset>
           </property>
           <property name="class" stdset="0">
            <string>tab_button</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="b_tabicon">
           <property name="text">
            <string/>
           </property>
           <property name="class" stdset="0">
            <string>tab_button</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="l_filename">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QFrame" name="fr_mainbutton">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_10">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QPushButton" name="b_main">
           <property name="minimumSize">
            <size>
             <width>42</width>
             <height>24</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Main Menu</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/menu.png</normaloff>img/menu.png</iconset>
           </property>
           <property name="class" stdset="0">
            <string>main_menu</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>6</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="b_show_tabs">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/tri_down.png</normaloff>img/tri_down.png</iconset>
           </property>
           <property name="class" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QFrame" name="fr_topleft">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>6</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <item row="0" column="4">
          <widget class="QFrame" name="fr_find">
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <property name="margin">
             <number>0</number>
            </property>
            <property name="spacing">
             <number>0</number>
            </property>
            <item row="0" column="3">
             <widget class="QFrame" name="fr_find_2">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <property name="class" stdset="0">
               <string>toolbarLineEdit</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_9">
               <property name="margin">
                <number>0</number>
               </property>
               <property name="spacing">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QLineEdit" name="le_find">
                 <property name="minimumSize">
                  <size>
                   <width>150</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background:transparent;
border:0px;</string>
                 </property>
                 <property name="placeholderText">
                  <string>find</string>
                 </property>
                 <property name="class" stdset="0">
                  <string>toolbar</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QPushButton" name="b_find">
                 <property name="maximumSize">
                  <size>
                   <width>32</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
background:transparent;
border-top-right-radius:3px;
border-bottom-right-radius:3px;
}
QPushButton:hover,QToolButton:hover {
	background-color: qlineargradient(spread:pad, x1:1, y1:1, x2:1, y2:0, stop:0 rgba(87, 87, 87, 150), stop:1 rgba(255, 255, 255, 150));
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="icon">
                  <iconset>
                   <normaloff>img/forward.png</normaloff>img/forward.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_goto">
              <property name="minimumSize">
               <size>
                <width>40</width>
                <height>24</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>goto</string>
              </property>
              <property name="class" stdset="0">
               <string>toolbar</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_8">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>10</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QPushButton" name="b_toggle_find">
           <property name="toolTip">
            <string>Run</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/search.png</normaloff>img/search.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="class" stdset="0">
            <string>toolbar toolbar-individual</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QFrame" name="fr_toolbar">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="5">
          <widget class="QPushButton" name="b_unindent">
           <property name="toolTip">
            <string>Unindent</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/indent_remove.png</normaloff>img/indent_remove.png</iconset>
           </property>
           <property name="class" stdset="0">
            <string>toolbar toolbar-middle</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QPushButton" name="b_indent">
           <property name="toolTip">
            <string>Indent</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/indent.png</normaloff>img/indent.png</iconset>
           </property>
           <property name="class" stdset="0">
            <string>toolbar toolbar-left</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="6">
          <widget class="QPushButton" name="b_comment">
           <property name="toolTip">
            <string>Comment/Uncomment</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/comment.png</normaloff>img/comment.png</iconset>
           </property>
           <property name="class" stdset="0">
            <string>toolbar toolbar-middle</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="b_new">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/new.png</normaloff>img/new.png</iconset>
           </property>
           <property name="class" stdset="0">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="8">
          <widget class="QPushButton" name="b_run">
           <property name="toolTip">
            <string>Run</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/tri_right.png</normaloff>img/tri_right.png</iconset>
           </property>
           <property name="class" stdset="0">
            <string>toolbar toolbar-individual</string>
           </property>
          </widget>
         </item>
         <item row="0" column="7">
          <widget class="QPushButton" name="b_color_picker">
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Insert rgb color&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/color_swatch.png</normaloff>img/color_swatch.png</iconset>
           </property>
           <property name="class" stdset="0">
            <string>toolbar toolbar-right</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="b_save">
           <property name="toolTip">
            <string>Save</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>img/save.png</normaloff>img/save.png</iconset>
           </property>
           <property name="class" stdset="0">
            <string>toolbar toolbar-individual</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="fr_tabs">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="l_statusbar">
     <property name="styleSheet">
      <string notr="true">QLabel {
padding-left:2px;
padding-top:2px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>le_find</sender>
   <signal>returnPressed()</signal>
   <receiver>b_find</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>794</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b_toggle_find</sender>
   <signal>toggled(bool)</signal>
   <receiver>fr_find</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>818</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>678</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
